integration/configs/background_replacement_test.config.json

```diff
@@ -0,0 +1,36 @@
+{
```

Comment:
Instead of creating a new config, can we just group blur and replacement together similar to VF:
https://github.com/aws/amazon-chime-sdk-js/blob/main/integration/configs/integrationTest/voice_focus_test.config.json#L7

Reply:
yes. I have updated and put the tests in a single file